src/openforms/submissions/tests/test_api_throttling.py

```diff
@@ -0,0 +1,97 @@
+import datetime
+from zoneinfo import ZoneInfo
+
+from django.conf import settings
+from django.core.cache import cache
+
+from freezegun import freeze_time
+from rest_framework import status
+from rest_framework.reverse import reverse
+from rest_framework.test import APITestCase
+
+from .factories import SubmissionFactory
+from .mixins import SubmissionsMixin
+
+
+class APIThrottlingTest(SubmissionsMixin, APITestCase):
+    def setUp(self):
+        super().setUp()
+        cache.clear()
+
+        throttles = settings.REST_FRAMEWORK["DEFAULT_THROTTLE_RATES"]
+        throttles["submit"] = "10/minute"
+        throttles["pause"] = "3/minute"
```

Comment:
this kind of setting mutation breaks test isolation, you _have_ to use `django.test.override_settings` so that settings can be restored after a test(case)